chapter01/rwd/HelloWord/src/Exercice1.java

```diff
@@ -0,0 +1,20 @@
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+
+import com.helloworld.Printer;
+
+public class Exercice1 {
+
+	public static void main(String[] args) {
+		ApplicationContext context = new ClassPathXmlApplicationContext("resources/beans.xml");
+
+		Printer helloEnlgish = (Printer) context.getBean("helloenlgish");
+		Printer helloSpanish = (Printer) context.getBean("hellospanish");
+
+		System.out.println(helloEnlgish.print());
+		System.out.println(helloSpanish.print());
+
+		((ClassPathXmlApplicationContext) context).close();
```

Comment:
I think, it is a bad thing to close the context unless it is needed. There is a danger of destroying all the beans.
If we want any clean better with `registerShutdownHook` may be it is covered later in the book.